models/Task.py

```diff
@@ -0,0 +1,25 @@
+from agents import ContextAgent
+from helpers import OpenAIHelper
+
+class Task:
+    def __init__(self, id: int, name: str, context_agent: ContextAgent):
+        self.id = id
+        self.name = name
+        self.result = None
+        self.context_agent = context_agent
+        self.open_ai_helper = OpenAIHelper()
+        self.objective = self.context_agent.get_objective()
+        if not self.name:
+            raise ValueError("Task name cannot be empty")
+
+    def print_self(self):
+        print(self.to_string())
+
+    def to_string(self):
+        return f"{self.id}\t{self.name}\t{self.result}"
```

Comment:
isn't the canonical/pythonic way of doing this `def __str__(self)`? then wherever else can just `print(task)` (or maybe `print(str(task))`)